diff -u expat-1.95.8-ori/expat_external.h expat/expat_external.h
--- expat-1.95.8-ori/expat_external.h	2004-05-06 15:55:49.000000000 +0200
+++ expat/expat_external.h	2004-08-07 14:06:08.000000000 +0200
@@ -4,10 +4,6 @@
 
 /* External API definitions */
 
-#if defined(_MSC_EXTENSIONS) && !defined(__BEOS__) && !defined(__CYGWIN__)
-#define XML_USE_MSC_EXTENSIONS 1
-#endif
-
 /* Expat tries very hard to make the API boundary very specifically
    defined.  There are two macros defined to control this boundary;
    each of these can be defined before including this header to
@@ -31,11 +27,6 @@
    system headers may assume the cdecl convention.
 */
 #ifndef XMLCALL
-#if defined(XML_USE_MSC_EXTENSIONS)
-#define XMLCALL __cdecl
-#elif defined(__GNUC__) && defined(__i386)
-#define XMLCALL __attribute__((cdecl))
-#else
 /* For any platform which uses this definition and supports more than
    one calling convention, we need to extend this definition to
    declare the convention used on that platform, if it's possible to
@@ -47,21 +38,9 @@
    platform's malloc() implementation.
 */
 #define XMLCALL
-#endif
 #endif  /* not defined XMLCALL */
 
 
-#if !defined(XML_STATIC) && !defined(XMLIMPORT)
-#ifndef XML_BUILDING_EXPAT
-/* using Expat from an application */
-
-#ifdef XML_USE_MSC_EXTENSIONS
-#define XMLIMPORT __declspec(dllimport)
-#endif
-
-#endif
-#endif  /* not defined XML_STATIC */
-
 /* If we didn't define it above, define it away: */
 #ifndef XMLIMPORT
 #define XMLIMPORT
Only in expat: expatcfg.h
diff -u expat-1.95.8-ori/internal.h expat/internal.h
--- expat-1.95.8-ori/internal.h	2003-09-28 23:57:22.000000000 +0200
+++ expat/internal.h	2004-08-07 14:03:15.000000000 +0200
@@ -20,54 +20,7 @@
          and therefore subject to change.
 */
 
-#if defined(__GNUC__) && defined(__i386__)
-/* We'll use this version by default only where we know it helps.
-
-   regparm() generates warnings on Solaris boxes.   See SF bug #692878.
-
-   Instability reported with egcs on a RedHat Linux 7.3.
-   Let's comment out:
-   #define FASTCALL __attribute__((stdcall, regparm(3)))
-   and let's try this:
-*/
-#define FASTCALL __attribute__((regparm(3)))
-#define PTRFASTCALL __attribute__((regparm(3)))
-#endif
-
-/* Using __fastcall seems to have an unexpected negative effect under
-   MS VC++, especially for function pointers, so we won't use it for
-   now on that platform. It may be reconsidered for a future release
-   if it can be made more effective.
-   Likely reason: __fastcall on Windows is like stdcall, therefore
-   the compiler cannot perform stack optimizations for call clusters.
-*/
-
-/* Make sure all of these are defined if they aren't already. */
-
-#ifndef FASTCALL
 #define FASTCALL
-#endif
-
-#ifndef PTRCALL
 #define PTRCALL
-#endif
-
-#ifndef PTRFASTCALL
 #define PTRFASTCALL
-#endif
-
-#ifndef XML_MIN_SIZE
-#if !defined(__cplusplus) && !defined(inline)
-#ifdef __GNUC__
-#define inline __inline
-#endif /* __GNUC__ */
-#endif
-#endif /* XML_MIN_SIZE */
 
-#ifdef __cplusplus
-#define inline inline
-#else
-#ifndef inline
-#define inline
-#endif
-#endif
diff -u expat-1.95.8-ori/xmlparse.c expat/xmlparse.c
--- expat-1.95.8-ori/xmlparse.c	2004-07-23 04:02:41.000000000 +0200
+++ expat/xmlparse.c	2004-08-07 14:03:49.000000000 +0200
@@ -8,13 +8,7 @@
 
 #define XML_BUILDING_EXPAT 1
 
-#ifdef COMPILED_FROM_DSP
-#include "winconfig.h"
-#elif defined(MACOS_CLASSIC)
-#include "macconfig.h"
-#elif defined(HAVE_EXPAT_CONFIG_H)
-#include <expat_config.h>
-#endif /* ndef COMPILED_FROM_DSP */
+#include "expatcfg.h"
 
 #include "expat.h"
 
diff -u expat-1.95.8-ori/xmlrole.c expat/xmlrole.c
--- expat-1.95.8-ori/xmlrole.c	2004-05-06 15:55:49.000000000 +0200
+++ expat/xmlrole.c	2004-08-07 14:03:15.000000000 +0200
@@ -4,15 +4,7 @@
 
 #include <stddef.h>
 
-#ifdef COMPILED_FROM_DSP
-#include "winconfig.h"
-#elif defined(MACOS_CLASSIC)
-#include "macconfig.h"
-#else
-#ifdef HAVE_EXPAT_CONFIG_H
-#include <expat_config.h>
-#endif
-#endif /* ndef COMPILED_FROM_DSP */
+#include "expatcfg.h"
 
 #include "expat_external.h"
 #include "internal.h"
diff -u expat-1.95.8-ori/xmltok.c expat/xmltok.c
--- expat-1.95.8-ori/xmltok.c	2004-05-06 15:55:49.000000000 +0200
+++ expat/xmltok.c	2004-08-07 14:03:15.000000000 +0200
@@ -4,15 +4,7 @@
 
 #include <stddef.h>
 
-#ifdef COMPILED_FROM_DSP
-#include "winconfig.h"
-#elif defined(MACOS_CLASSIC)
-#include "macconfig.h"
-#else
-#ifdef HAVE_EXPAT_CONFIG_H
-#include <expat_config.h>
-#endif
-#endif /* ndef COMPILED_FROM_DSP */
+#include "expatcfg.h"
 
 #include "expat_external.h"
 #include "internal.h"
