diff -u --recursive --new-file expat-1.95.6/expat.h expat-int/expat.h
--- expat-1.95.6/expat.h	2003-01-16 23:03:42.000000000 +0100
+++ expat-int/expat.h	2003-06-15 17:22:55.000000000 +0200
@@ -103,6 +103,13 @@
   XML_CQUANT_PLUS
 };
 
+enum XML_Status {
+  XML_STATUS_ERROR = 0,
+#define XML_STATUS_ERROR XML_STATUS_ERROR
+  XML_STATUS_OK = 1
+#define XML_STATUS_OK XML_STATUS_OK
+};
+
 /* If type == XML_CTYPE_EMPTY or XML_CTYPE_ANY, then quant will be
    XML_CQUANT_NONE, and the other fields will be zero or NULL.
    If type == XML_CTYPE_MIXED, then quant will be NONE or REP and
@@ -732,12 +739,6 @@
 
    Otherwise, the #define hackery is quite ugly and would have been dropped.
 */
-enum XML_Status {
-  XML_STATUS_ERROR = 0,
-#define XML_STATUS_ERROR XML_STATUS_ERROR
-  XML_STATUS_OK = 1
-#define XML_STATUS_OK XML_STATUS_OK
-};
 
 XMLPARSEAPI(enum XML_Status)
 XML_Parse(XML_Parser parser, const char *s, int len, int isFinal);
diff -u --recursive --new-file expat-1.95.6/intconfig.h expat-int/intconfig.h
--- expat-1.95.6/intconfig.h	1970-01-01 01:00:00.000000000 +0100
+++ expat-int/intconfig.h	2003-07-14 23:00:10.000000000 +0200
@@ -0,0 +1,25 @@
+#ifndef INTCONFIG_H
+#define INTCONFIG_H
+
+#include <stdlib.h>
+#include <string.h>
+
+#define XML_NS 1
+#define XML_DTD 1
+#define XML_CONTEXT_BYTES 1024
+
+#if defined(USE_LSB) || defined(LSB_FIRST)
+#define BYTEORDER 1234
+#else
+#define BYTEORDER 4321
+#endif
+
+#define HAVE_MEMMOVE
+
+#define XMLPARSEAPI(type) type
+
+#include "expat.h"
+
+#undef XMLPARSEAPI
+
+#endif
diff -u --recursive --new-file expat-1.95.6/internal.h expat-int/internal.h
--- expat-1.95.6/internal.h	2002-09-20 05:42:43.000000000 +0200
+++ expat-int/internal.h	2003-07-14 23:02:36.000000000 +0200
@@ -20,54 +20,7 @@
          and therefore subject to change.
 */
 
-#if defined(__GNUC__)
-/* Instability reported with egcs on a RedHat Linux 7.3.
-   Let's comment it out:
-   #define FASTCALL __attribute__((stdcall, regparm(3)))
-   and let's try this:
-*/
-#define FASTCALL __attribute__((regparm(3)))
-#define PTRCALL
-#define PTRFASTCALL __attribute__((regparm(3)))
-
-#elif defined(WIN32)
-/* Using __fastcall seems to have an unexpected negative effect under
-   MS VC++, especially for function pointers, so we won't use it for
-   now on that platform. It may be reconsidered for a future release
-   if it can be made more effective.
-   Likely reason: __fastcall on Windows is like stdcall, therefore
-   the compiler cannot perform stack optimizations for call clusters.
-*/
-#define FASTCALL
-#define PTRCALL
-#define PTRFASTCALL
-
-#endif
-
-#ifndef FASTCALL
 #define FASTCALL
-#endif
-
-#ifndef PTRCALL
 #define PTRCALL
-#endif
-
-#ifndef PTRFASTCALL
 #define PTRFASTCALL
-#endif
-
-#ifndef XML_MIN_SIZE
-#if !defined(__cplusplus) && !defined(inline)
-#ifdef __GNUC__
-#define inline __inline
-#endif /* __GNUC__ */
-#endif
-#endif /* XML_MIN_SIZE */
 
-#ifdef __cplusplus
-#define inline inline
-#else
-#ifndef inline
-#define inline
-#endif
-#endif
diff -u --recursive --new-file expat-1.95.6/intexpat.h expat-int/intexpat.h
--- expat-1.95.6/intexpat.h	1970-01-01 01:00:00.000000000 +0100
+++ expat-int/intexpat.h	2003-08-20 22:44:28.000000000 +0200
@@ -0,0 +1,6 @@
+#ifndef INTEXPAT_H
+#define INTEXPAT_H
+
+#include "expat.h"
+
+#endif
diff -u --recursive --new-file expat-1.95.6/xmlparse.c expat-int/xmlparse.c
--- expat-1.95.6/xmlparse.c	2003-01-28 06:45:10.000000000 +0100
+++ expat-int/xmlparse.c	2003-06-15 17:22:55.000000000 +0200
@@ -5,32 +5,7 @@
 #include <stddef.h>
 #include <string.h>                     /* memset(), memcpy() */
 
-#ifdef COMPILED_FROM_DSP
-
-#include "winconfig.h"
-#define XMLPARSEAPI(type) type __cdecl
-#include "expat.h"
-#undef XMLPARSEAPI
-
-#elif defined(MACOS_CLASSIC)
-
-#include "macconfig.h"
-#include "expat.h"
-
-#else
-
-#include <expat_config.h>
-
-#ifdef __declspec
-#define XMLPARSEAPI(type) type __cdecl
-#endif
-
-#include "expat.h"
-
-#ifdef __declspec
-#undef XMLPARSEAPI
-#endif
-#endif /* ndef COMPILED_FROM_DSP */
+#include "intconfig.h"
 
 #ifdef XML_UNICODE
 #define XML_ENCODE_MAX XML_UTF16_ENCODE_MAX
diff -u --recursive --new-file expat-1.95.6/xmlrole.c expat-int/xmlrole.c
--- expat-1.95.6/xmlrole.c	2002-09-20 05:42:43.000000000 +0200
+++ expat-int/xmlrole.c	2003-06-15 17:22:55.000000000 +0200
@@ -2,13 +2,7 @@
    See the file COPYING for copying permission.
 */
 
-#ifdef COMPILED_FROM_DSP
-#include "winconfig.h"
-#elif defined(MACOS_CLASSIC)
-#include "macconfig.h"
-#else
-#include <expat_config.h>
-#endif /* ndef COMPILED_FROM_DSP */
+#include "intconfig.h"
 
 #include "internal.h"
 #include "xmlrole.h"
diff -u --recursive --new-file expat-1.95.6/xmltok.c expat-int/xmltok.c
--- expat-1.95.6/xmltok.c	2002-12-14 04:12:18.000000000 +0100
+++ expat-int/xmltok.c	2003-06-15 17:22:55.000000000 +0200
@@ -2,13 +2,7 @@
    See the file COPYING for copying permission.
 */
 
-#ifdef COMPILED_FROM_DSP
-#include "winconfig.h"
-#elif defined(MACOS_CLASSIC)
-#include "macconfig.h"
-#else
-#include <expat_config.h>
-#endif /* ndef COMPILED_FROM_DSP */
+#include "intconfig.h"
 
 #include "internal.h"
 #include "xmltok.h"
